File: net-examples/Aspose.Font.Examples/Licensing/SetLicenseFromFile.cs
using System;
using System.IO;

namespace Aspose.Font.Examples.Licensing
{
    class SetLicenseFromFile : BaseExamples
    {
        public override void Run()
        {
            Console.WriteLine("Run Set License from File example");

            //ExampleStart: 1
            string licensePath = Path.Combine(LicenseDir, "Aspose.Font.lic");

            License license = new License();
            license.SetLicense(licensePath);
            //ExampleEnd: 1
        }
    }
}

File: cpp-examples/Aspose.Font.Examples.CPP/source/Licensing/SetLicenseFromFile.cpp
#include "aspose_pch.h"
#include "SetLicenseFromFile.h"

#include <system/string.h>
#include <system/io/path.h>
#include <system/console.h>
#include <Aspose.Font.Cpp/Licensing/FontLicense.h>

namespace Aspose {

namespace Font {

namespace Examples {

namespace Licensing {

RTTI_INFO_IMPL_HASH(694846920u, ::Aspose::Font::Examples::Licensing::SetLicenseFromFile, ThisTypeBaseTypesInfo);

void SetLicenseFromFile::Run()
{
    System::Console::WriteLine(u"Run Set License from File example");
    
    //ExampleStart: 1
    System::String licensePath = System::IO::Path::Combine(get_LicenseDir(), u"Aspose.Font.CPP.lic");
    
    System::SharedPtr<License> license = System::MakeObject<License>();
    license->SetLicense(licensePath);
    //ExampleEnd: 1
}

} // namespace Licensing
} // namespace Examples
} // namespace Font
} // namespace Aspose
